utils/k8s/k8s.go

```diff
@@ -660,3 +661,21 @@ func (k8s K8S) BlockUntilPodIsUpOrTimeout(pod *core_v1.Pod, timeout time.Duratio
 
 	return k8s.BlockUntilPodIsUpWithContext(ctx, pod)
 }
+
+// GetStorageClass returns the StorageClass object for given storageClassName.
+func (k8s K8S) GetStorageClass(storageClassName string) (*storage_v1.StorageClass, error) {
+	storageClassClient := k8s.Clientset.StorageV1().StorageClasses()
+	return storageClassClient.Get(storageClassName, meta_v1.GetOptions{})
+}
+
+// ListStorageClasses returns all the StorageClass object.
+func (k8s K8S) ListStorageClasses() (*storage_v1.StorageClassList, error) {
+	storageClassClient := k8s.Clientset.StorageV1().StorageClasses()
+	return storageClassClient.List(meta_v1.ListOptions{})
+}
+
+// DeleteStorageClass deletes the StorageClass object for given storageClassName.
```

Comment:
// DeleteStorageClass deletes the StorageClass object of given storageClassName.